backend/app/services/adapters/task_kinds.py

```
@@ -2747,6 +2775,27 @@ def _get_tasks_related_data_batch(
+        """
+        try:
+            labels = task_crd.metadata.labels
+            if not labels:
+                return False
+
+            # Check for background task indicators
+            return (
+                labels.get("source") == "background_executor"
+                or labels.get("type") == "background"
+            )
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)